src/main/java/org/mskcc/cbio/portal/dao/DaoStructuralVariant.java

```diff
@@ -151,6 +153,33 @@ public static void addStructuralVariantToBulkLoader(StructuralVariant structural
         }
     }
 
+    public static void deleteStructuralVariants(int geneticProfileId, Set<Integer> sampleIds) throws DaoException {
+        Connection con = null;
+        PreparedStatement pstmt = null;
+        ResultSet rs = null;
+        try {
+            con = JdbcUtil.getDbConnection(DaoGene.class);
+            pstmt = con.prepareStatement("DELETE structural_variant, alteration_driver_annotation" +
```

Comment:
`alteration_driver_annotation`: how is this linked to `structural_variant`? It seems to have been introduced for CNA data only:
![image](https://github.com/cBioPortal/cbioportal-core/assets/2900303/054f52de-b7b5-4747-82e5-f9ccc1db4c3f)

Have you found evidence it is used for SV data as well?

Reply:
`alteration_driver_annotation ` is liked to `structural_variant ` with `SAMPLE_ID` and `GENETIC_PROFILE_ID`.

Here is why I thought the table is used for this data type as well:
https://github.com/cBioPortal/cbioportal-core/blob/1e8f32b11870779dd529fb642c468250e055b875/src/main/java/org/mskcc/cbio/portal/dao/DaoStructuralVariant.java#L142

I did not see any input files exercising this though.

Let me know if we should remove this code.

Reply:
It seems it was intentional - see this PR   description by @pvannierop (title is misleading though) https://github.com/cBioPortal/cbioportal/pull/7894